tests/test_freqtradebot.py

```diff
@@ -1207,6 +1210,7 @@ def test_handle_stoploss_on_exchange_trailing(mocker, default_conf, fee,
     patch_get_signal(freqtrade)
 
     freqtrade.enter_positions()
+    # TODO-lev: Get this trade switched to the usdt trades
```

Comment:
I'm not sure how to get this test to pass because I don't know how to set the trades inside `Trade.query`

Right now the old btc price trades are returned from `Trade.query`, and I would like to update them so that the usdt price trades are returned, but it's hard to locate where the program actually creates the mock trades

Reply:
whenever `enter_positions()` is called, it's not mocked trades, but "kindof real" trades.

To have them use different currencies, you'll have to change stake_currency in default_conf and use a different pairlist. Obviously the results (create_order, fetch_order, ...) need to align to this as well.
That'll however impact the whole testsuite - so you'll need to carefully consider if that's something you wanna do at this point (i'd try to avoid creating a second default_config).

The asset name that's used is not really important in my opinion (it'll be the first (few) of the pair_whitelist) - but the values should be "normal" values.

Reply:
> whenever `enter_positions()` is called, it's not mocked trades, but "kindof real" trades.
> 
> To have them use different currencies, you'll have to change stake_currency in default_conf and use a different pairlist. Obviously the results (create_order, fetch_order, ...) need to align to this as well.
> That'll however impact the whole testsuite - so you'll need to carefully consider if that's something you wanna do at this point (i'd try to avoid creating a second default_config).
> 
> The asset name that's used is not really important in my opinion (it'll be the first (few) of the pair_whitelist) - but the values should be "normal" values.

it's the prices and amounts that I want to switch, the amount is something like `0.017300332` or something, and I'd like to switch it to `30.0` or something like that

Reply:
there's a whole range of settings playing together - it's not that different for tests than if you configure a real configuration.
if some parts don't align (stake_amount for example) - you'll get odd results.

Reply:
> there's a whole range of settings playing together - it's not that different for tests than if you configure a real configuration.
> if some parts don't align (stake_amount for example) - you'll get odd results.

Im just wondering where these trades are created, I've tried to backtrack through the function calls, but can't find it

Reply:
enter_positions() :)

it'll create the trades as part of it's functionality - based on the strategy / exchange (moked) responses.

Reply:
I'm really confused where the value `0.01105263` is coming from, that is the correct value that amount is equal to for `stoploss_order_mock.assert_called_once_with`, what can i mess with to change this value